### PR TITLE
feat: EC Page Header

### PR DESCRIPTION
**Pull request by nimek2**
# PR description

Please drop a few lines about the PR: what it does, how to test it, etc.

## QA Checklist

In order to ensure a safe and quick review, please check that your PR follow those guidelines:

- [x] I have put the vanilla component as `devDependencies`
- [x] I have put the specs package as `devDependencies`
- [x] I have added the components directly used in the twig file (with `include` or `embed`) as `dependencies`
- [x] My component is listed in `@ecl-twig/ec-components`'s `dependencies`
- [x] My variables naming follow the guidelines (snake case for twig)
- [x] I have provided tests
- [x] I have provided documentation (for the "notes" tab)
- [x] If my local `yarn.lock` contains changes, I have committed it
- [x] I have given my PR the proper label (`pr: review needed` to indicate that I'm done and now waiting for a review ,`pr: wip` to indicate that I'm actively working on it ...)

### CODE REVIEW

**Comment by degliwe** [comment]
@nimek2 please read and complete the checklist on the PR description. Please put a label to your PR so that we know if it's in progress or waiting for a review.

File: src/ec/packages/ec-component-page-header/page-header.html.twig [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+      {%- include '../ec-component-breadcrumb/breadcrumb.html.twig' with breadcrumb|merge({ extra_classes: 'ecl-page-header__breadcrumb' }) -%}
+    {%- endif -%}
+    {%- if _meta is not empty -%}
+      <div class="ecl-page-header__meta-list">{{ _meta }}</div>
+    {%- endif -%}
+    <h1 class="ecl-page-header__title">{{ _title }}</h1>
+    {%- if _description is not empty -%}
+      <p class="ecl-page-header__description">{{ _description }}</p>
+    {%- endif -%}
+    {%- if infos is defined and infos not empty -%}
```
Comment (kalinchernev) [inline src/ec/packages/ec-component-page-header/page-header.html.twig]: Might worth adding a check for `iterable`, as the React one seems to be ready to accept not only arrays https://github.com/ec-europa/europa-component-library/blob/v2-dev/src/systems/ec/implementations/react/packages/page-header/src/PageHeader.jsx#L17

I am not 100% sure it's necessary though, as in this implementation there is a default of empty array.

File: src/ec/packages/ec-component-page-header/page-header.story.js [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+      );
+
+      return pageHeader(sampleData);
+    },
+    {
+      notes: { markdown: pageHeaderDocs },
+    }
+  )
+  .add(
+    'events',
```
Comment (kalinchernev) [inline src/ec/packages/ec-component-page-header/page-header.story.js]: Please provide knobs which are given in the other implementation as well https://v2--europa-component-library.netlify.com/storybook/ec/index.html?selectedKind=PageHeader&selectedStory=events&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybooks%2Fstorybook-addon-knobs

File: src/ec/packages/ec-component-page-header/page-header.story.js [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+      const sampleData = preparePageHeaderData(pageHeaderDataEvents);
+
+      return pageHeader(sampleData);
+    },
+    {
+      notes: { markdown: pageHeaderDocs },
+    }
+  )
+  .add(
+    'events-description',
```
Comment (kalinchernev) [inline src/ec/packages/ec-component-page-header/page-header.story.js]: A few missing knobs: https://v2--europa-component-library.netlify.com/storybook/ec/index.html?selectedKind=PageHeader&selectedStory=events-description&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybooks%2Fstorybook-addon-knobs

File: src/ec/packages/ec-component-page-header/page-header.test.js [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+import breadcrumbDataSimple from '@ecl/ec-specs-breadcrumb/demo/data-simple';
+
+import pageHeaderDataTitle from '@ecl/ec-specs-page-header/demo/data-title';
+import pageHeaderDataTitleDescription from '@ecl/ec-specs-page-header/demo/data-title-description';
+import pageHeaderDataMetaTitle from '@ecl/ec-specs-page-header/demo/data-meta-title';
+import pageHeaderDataMetaTitleDescription from '@ecl/ec-specs-page-header/demo/data-meta-title-description';
+import pageHeaderDataEvents from '@ecl/ec-specs-page-header/demo/data-events';
+import pageHeaderDataEventsDescription from '@ecl/ec-specs-page-header/demo/data-events-description';
+
+function formatBreadcrumbLink(l) {
```
[the remark this answers is not in the document]
Comment (hapertown) [reply src/ec/packages/ec-component-page-header/page-header.test.js]: Reason -> https://github.com/ec-europa/ecl-twig/pull/37#discussion_r261197632

File: src/ec/packages/ec-components/package.json [hunk 5]
```diff
@@ -25,7 +24,9 @@
     "@ecl-twig/ec-component-social-media-follow": "^0.0.1-alpha",
     "@ecl-twig/ec-component-social-media-share": "^0.0.1-alpha",
     "@ecl-twig/ec-component-tag": "^0.0.1-alpha",
-    "@ecl-twig/ec-component-text-input": "^0.0.1-alpha"
+    "@ecl-twig/ec-component-text-input": "^0.0.1-alpha",
+    "@ecl-twig/ec-component-breadcrumb": "^0.0.1-alpha",
```
[the remark this answers is not in the document]
Comment (hapertown) [reply src/ec/packages/ec-components/package.json]: Removed in https://github.com/ec-europa/ecl-twig/pull/53/commits/2b6a9d0c4224fba2d0e32ba625329bfdde721154

File: src/ec/packages/ec-component-page-header/page-header.story.js (hunk 2 above)
Comment (hapertown) [reply src/ec/packages/ec-component-page-header/page-header.story.js]: Synchronized knobs 1:1 with documentation in https://github.com/ec-europa/ecl-twig/pull/53/commits/ca4852efa2fa10288e41088c11f32ed3aaeb7ce5

File: src/ec/packages/ec-component-page-header/page-header.story.js (hunk 3 above)
Comment (hapertown) [reply src/ec/packages/ec-component-page-header/page-header.story.js]: See https://github.com/ec-europa/ecl-twig/pull/53#discussion_r263696634

File: src/ec/packages/ec-component-page-header/page-header.html.twig (hunk 1 above)
Comment (hapertown) [reply src/ec/packages/ec-component-page-header/page-header.html.twig]: Fixed in https://github.com/ec-europa/ecl-twig/pull/53/commits/ca4852efa2fa10288e41088c11f32ed3aaeb7ce5

File: src/ec/packages/ec-component-page-header/page-header.html.twig [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+
+{% if extra_attributes is defined and extra_attributes is not empty and extra_attributes is iterable %}
+  {% for attr in extra_attributes %}
+    {% set _extra_attributes = _extra_attributes ~ ' ' ~ attr.name ~ '="' ~ attr.value ~ '"' %}
+  {% endfor %}
+{% endif %}
+
+{# Print the result #}
+<div class="{{ _css_class }}"{{ _extra_attributes }}>
+  <div class="ecl-container">
```
[the remark this answers is not in the document]
Comment (hapertown) [reply src/ec/packages/ec-component-page-header/page-header.html.twig]: Please check if I added it correctly in https://github.com/ec-europa/ecl-twig/pull/53/commits/ca4852efa2fa10288e41088c11f32ed3aaeb7ce5

[the remark this answers is not in the document]
Comment (yhuard) [reply src/ec/packages/ec-component-page-header/page-header.html.twig]: LGTM

**Pull request merged**